pkg/domain/infra/runtime_libpod.go

```diff
@@ -126,12 +127,17 @@ func getRuntime(ctx context.Context, fs *flag.FlagSet, opts *engineOpts) (*libpo
 		storageSet = true
 	}
 
-	if fs.Changed("root") {
+	gr, _ := fs.GetString("graph-root")
+	if fs.Changed("root") || len(gr) > 0 {
```

Comment:
I don't see the value of gr being used?

Reply:
rgr is just being used below in `if fs.Changed("root") || len(gr) > 0 {` the issue is I cannot just do `fs.Changed` since the flag will not always exist at runtime.

Reply:
Huh?  fs.Changed("root") the --root not existing?

Reply:
no `--graph-root` is a non-global system reset flag but many different commands go through `getRuntime`. If i always grab the flag via `fs.Changed` and it does not exist in the command being run, we error out

Reply:
Using `fs.Lookup("graph-root")` might be clearer. It returns nil if the flag is not in the FlagSet.